snapcraft/common.py

```diff
@@ -100,10 +100,16 @@ def get_arch_triplet():
         return _DEB_TRANSLATIONS[platform.machine()]['triplet']
     except KeyError:
         raise EnvironmentError(
-            '{} is not supported, please log a bug at'
+            '{} is not supported, please log a bug at '
             'https://bugs.launchpad.net/snapcraft'.format(platform.machine()))
```

Comment:
ooh :-) can I suggest you change the link to go to e.g. `https://bugs.launchpad.net/snapcraft/+filebug?field.title=please+add+support+for+{1}` ?